picard/util/tags.py

```diff
@@ -136,6 +136,7 @@
     'totaltracks': N_('Total Tracks'),
     'tracknumber': N_('Track Number'),
     'website': N_('Artist Website'),
+    'user_website': N_('User Defined Website'),
```

Comment:
What about all the other non-ID3 formats - how will they handle this completely new tag?????

Reply:
Yes, this needs to be defined. We shouldn't just write `USER_WEBSITE` to Vorbis tags, this seems odd. I actually wouldn't expect a fix for the LINK frame issue to introduce a new tag. If we add a new tag it should be well handled from the start. If in doubt I would rather not write a new tag to some formats initially until we have figured out a proper mapping.